packages/server-renderer/__tests__/render.spec.ts

```diff
@@ -128,14 +128,14 @@ function testRender(type: string, render: typeof renderToString) {
           await render(
             createApp(
               defineComponent({
-                extends: {
+                extends: defineComponent({
```

Comment:
Typescript has hard type inferring component options, I believe we should always enforce the usage of `defineComponent` in this ocasions